### PR TITLE
[ISV-749] Update the download_test_results function

### PR DESCRIPTION
**Pull request by haripate**

### CODE REVIEW

**Review by MarcinGinszt — APPROVED** [review]
LGTM

File: operatorcert/entrypoints/git_add_comment.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    if args.test_result == "success":
+        LOGGER.info("Tests passed successfully.")
+        sys.exit(1)
+    split_url = urllib.parse.urlparse(args.request_url).path.split("/")
+    # This will convert https://github.com/foo/bar/pull/202 to
+    # api url path /repos/foo/issues/
+    api_url = "{base}/repos/{package}/issues/{id}".format(
+        base="", package="/".join(split_url[1:3]), id=split_url[-1]
+    )
+    commentParamValue = f"""{args.comment_or_file}"""
```
[the remark this answers is not in the document]
Comment (haripate) [reply operatorcert/entrypoints/git_add_comment.py]: It has been removed.

File: operatorcert/__init__.py [hunk 2]
```diff
@@ -356,6 +356,8 @@ def download_test_results(args) -> Optional[str]:
         {
             "passed": query_results[0]["passed"],
             "results": query_results[0]["results"],
+            "pull_request": query_results[0]["pull_request"],
+            "test_library": query_results[0]["test_library"],
```
Comment (MarcinGinszt) [inline operatorcert/__init__.py:360]: Where is it needed?

**Comment by MarcinGinszt** [comment]
LGTM! :)

**Pull request merged**